src/pipecat/services/gladia/stt.py

```diff
@@ -561,15 +560,11 @@ async def _receive_task_handler(self):
                                 translation, "", time_now_iso8601(), translated_language
                             )
                         )
-
-                self.reset_watchdog()
```

Comment:
It does feels we need this one right ? Like, the task is completed, so we should reset the watchdog. Otherwise we are only going to do this internally in the `WatchdogAsyncIterator`, when we receive the next element.